doc/src/general-rules.md

```diff
@@ -92,3 +92,18 @@ members = [
 +   "rahulku"
 ]
 ```
+
+Committee members are expected to contribute by reviewing pull requests (all
+pull requests review approvals from at least two committee members before they
+can be merged).
+Reviews of solutions towards challenges should consider at least the following aspects:
+
+1. Does the pull request implement a solution that respects/meets the success
+   criteria of the challenge?
+2. Do the contracts and harnesses incorporate the safety conditions stated in
+   the documentation (from comments in the code and Rust book)? Note that we
+   currently focus on safety verification. Pre- and post-conditions towards
+   functional correctness are acceptable as long as they do not negatively
```

Comment:
IMO we should add something here about also not accepting functional correctness constraints that are overly long or complex--e.g., take a long time to verify, or are difficult enough to maintain that they're unlikely to get merged upstream.
(We can say something about how we know this is subjective, but to use your best judgment or something).

Reply:
I've tried to adopt your suggestion, could you please take another look?